src/main/resources/data/portalcubed/tags/blocks/bullet_hole_metal.json

```diff
@@ -43,21 +45,152 @@
     "minecraft:waxed_oxidized_cut_copper",
     "minecraft:waxed_oxidized_cut_copper_stairs",
     "minecraft:waxed_oxidized_cut_copper_slab",
+    {
+      "id": "minecraft:chiseled_copper",
+      "required": false
```

Comment:
pretty sure these don't need to be optional, was that tested

Reply:
fixed